mu.yaml

```diff
@@ -28,6 +28,11 @@ months:
     mday: 18
     year_ranges:
       - limited: [2022]
+  - name: Chinese Spring Festival
+    regions: [mu]
+    mday: 29
+    year_ranges:
+      - limited: [2025]
```

Comment:
Does this mean this holiday definition only works in 2025 and we'll have to update this in 2026? Does the holiday not happen every year with some kind of pattern?

Reply:
Unfortunately unsupported by the gem 
<img width="644" alt="image" src="https://github.com/user-attachments/assets/0f329e54-5307-43c1-861b-2cc3416b75ef" />